### PR TITLE
Kusto: Fix database ARM templates not being generated

### PR DESCRIPTION
**Pull request by radennis**
Currently cluster resources are being generated successfully in the portal but database sub resources are not. This changes tries to fix it.

### CODE REVIEW

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2014-04-01-preview/Microsoft.Sql.json#/resourceDefinitions/administrators" },
                   { "$ref": "https://schema.management.azure.com/schemas/2014-04-01-preview/Microsoft.Sql.json#/resourceDefinitions/disasterrecoveryconfiguration" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-07-privatepreview/Microsoft.Kusto.json#/resourceDefinitions/clusters" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-09-07-privatepreview/Microsoft.Kusto.json#/resourceDefinitions/databases" },
```
Comment (TianoMS) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: >databases [](start = 141, length = 9)

update in 2015-01-01/deploymentTemplate.json as well?

Comment (radennis) [reply schemas/2014-04-01-preview/deploymentTemplate.json]: Done

File: tests/2017-09-07-privatepreview/Microsoft.Kusto.tests.json [hunk 2]
```diff
@@ -4,14 +4,14 @@
       "name": "clusters - Minimal",
       "definition": "https://schema.management.azure.com/schemas/2017-09-07-privatepreview/Microsoft.Kusto.json#/resourceDefinitions/clusters",
       "json": {
+		"name": "cluster1",
```
Comment (TianoMS) [inline tests/2017-09-07-privatepreview/Microsoft.Kusto.tests.json]: >		 [](start = 0, length = 2)

please use spaces instead of tabs..

**Comment by Tiano2017** [comment]
PR looks good. just one small issue on format.

File: tests/2017-09-07-privatepreview/Microsoft.Kusto.tests.json (hunk 2 above)
Comment (radennis) [reply tests/2017-09-07-privatepreview/Microsoft.Kusto.tests.json]: Done

**Pull request merged**